templates/cloud/cloud-computing.html

```diff
@@ -2,486 +2,702 @@
-      <p>In turn, cloud computing expands the capabilities provided by the hypervisor. The cloud software manages all the hypervisors and automates the VM provisioning process. Whenever a cloud user requests a VM with specific parameters through the self-service portal, it tries to find the least utilised hypervisor (unless configured otherwise), schedules VM creation on this hypervisor and returns VM details back to the user. Likewise, it automates the creation of other types of virtual resources, such as block storage devices, virtual networks, and more.</p>
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>What is cloud computing?</h1>
+      </div>
+      <div class="col">
+        <p>
+          Cloud computing is the on-demand delivery of IT resources over a network. In traditional data centres, compute and storage resources used to be allocated manually by a dedicated IT team. In the cloud, this process is fully automated, leading to increased agility and significant cost savings.
+        </p>
+        <p>
+          Cloud computing is a <em>de facto</em> standard for modern data centre implementation and a foundation for <a href="/blog/what-is-cloud-native">cloud-native application</a> deployments. It is used by the majority of organisations worldwide. There is no reason why yours shouldn’t be next.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a class="p-button--positive"
+             href="/engage/a-business-guide-to-cloud-migration">Get ready for cloud migration&nbsp;&rsaquo;</a>
+        </p>
+      </div>
     </div>
-  </div>
-</section>
-<section class="p-strip">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>Cloud computing benefits</h2>
-      <p class="u-sv3">Compared to legacy IT infrastructure based on mainframes, blades and "pet” servers, cloud computing transforms data centres into resource pools, providing end users with immediate access to hardware: <a href="https://maas.io">physical</a> or <a href="/openstack/features">virtual</a>. The primary benefits of cloud computing include:</p>
+    <div class="row">
+      <div class="p-image-container--cinematic u-hide--small is-highlighted">
+        <img class="p-image-container__image p-strip is-shallow"
+             src="https://assets.ubuntu.com/v1/667fa185-Artboard%20150@3x.png"
+             alt="MultiCloud"
```

Comment:
```suggestion
             alt="MultiCloud"
```
For an image this complicated you would need provide a more detailed alt text. Or, you can omit it completely, as the image offer zero information.